gen/testvars/fvRemoteId.yaml

```diff
@@ -0,0 +1,55 @@
+# Code generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+
+default:
+  annotation: "orchestrator:terraform"
+  description: ""
+  name: ""
+  name_alias: ""
+  owner_key: ""
+  owner_tag: ""
+  remote_ctx_pc_tag: ""
+  remote_pc_tag: ""
+  site_id: "0"
+
+datasource_non_existing:
+  site_id: "non_existing_site_id"
+
+datasource_required:
+  site_id: "test_site_id"
+
+resource_required:
+  site_id: "0"
+
+all:
+  annotation: "annotation"
+  description: "description"
+  name: "name"
+  name_alias: "name_alias"
+  owner_key: "owner_key"
+  owner_tag: "owner_tag"
+  remote_ctx_pc_tag: "remote_ctx_pc_tag"
+  remote_pc_tag: "remote_pc_tag"
```

Comment:
is apic allowing just regular strings for this? would have expected a pctag id here